e2e/rook-test-framework/transport/standalone_transport_client.go

```diff
@@ -0,0 +1,102 @@
+package transport
+
+import (
+	"bytes"
+	"errors"
+	"os/exec"
+	"syscall"
+)
+
+type standAloneTransportClient struct {
+}
+
+func CreateNewStandAloneTransportClient() *standAloneTransportClient {
+	return &standAloneTransportClient{}
+}
+
+func (k *standAloneTransportClient) ExecuteCmd(cmd []string) (stdout string, stderr string, err error) {
+	return "", "", errors.New("Not Implemented")
+}
+
+func (k *standAloneTransportClient) Execute(cmdArgs []string, optional []string) (stdout string, stderr string, exitCode int) {
+
+	var outbuf, errbuf bytes.Buffer
+	cmd := exec.Command("rook", cmdArgs...)
+	cmd.Stdout = &outbuf
+	cmd.Stderr = &errbuf
+
+	err := cmd.Run()
+	stdout = outbuf.String()
+	stderr = errbuf.String()
+
+	if err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			ws := exitError.Sys().(syscall.WaitStatus)
+			exitCode = ws.ExitStatus()
+		} else {
+			exitCode = defaultFailedCode
+			if stderr == "" {
+				stderr = err.Error() + stdout
+			}
+		}
+	} else {
+		ws := cmd.ProcessState.Sys().(syscall.WaitStatus)
+		exitCode = ws.ExitStatus()
+	}
+	return
+}
+
+func (k *standAloneTransportClient) Create(cmdArgs []string, optional []string) (stdout string, stderr string, exitCode int) {
+
+	var outbuf, errbuf bytes.Buffer
```

Comment:
this function looks identical to the `Execute` function above it and the `Delete` function below it.  consider refactoring to remove this duplication.